plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -998,4 +999,35 @@ protected void verifyTableNameLengthFailurePermissible(Throwable e)
     {
         assertThat(e).hasMessage("Table name must be shorter than or equal to '63' characters but got '64'");
     }
+
+    @Test(timeOut = 120_000, dataProvider = "fixedJoinDistributionTypes")
+    public void testJoinDynamicFilteringMultiJoin(JoinDistributionType joinDistributionType)
```

Comment:
We already have couple of `testJoinDynamicFilteringMultiJoin` methods. Do we need a next one?

If so, I would add a comment that this is the same method as XXX and we run it here because...

Reply:
removed it as it is redundant with the other one